src/main/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherTaxServiceImpl.java

```diff
@@ -80,4 +86,48 @@ protected boolean validateNRATaxInformation(DisbursementVoucherDocument document
         }
         return true;
     }
+    
+	public boolean isForeignVendorWithTaxReviewRequired(String payeeTypeCode,
```

Comment:
I would recommend adding the "@Override" annotation to any overridden or implemented methods, to help minimize errors when future code changes are made.